src/pages/PatientDashboard.js

```diff
@@ -1,3 +1,5 @@
+import React from "react"
```

Comment:
Is it necessary to have this? I don't think current version will give errors if it's not included

Reply:
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

> Upgrading to the new transform is completely optional, but it has a few benefits:
>
> - With the new transform, you can use JSX without importing React.
> - ...

Reply:
right
I installed react 16 for testing something, had to add this during the process

Reply:
will remove it